premailer/data/leftover_test.html

```diff
@@ -0,0 +1,20 @@
+<html>
+
+<head>
+    <style>
+        div {
+            background-color: red;
+            padding: 10px;
+        }
+
+        .a:hover {
+            background-color: green;
```

Comment:
Could you please add none more property here. Look like it may be a bug when we make string of all properties?
I unfortunately doesn't have a proper computer with Golang installed :-(

Reply:
Do you mean when we add another unmergable class or two parameters in the same class?
Something like this works? Although, class ```b``` remains on div element.
```css
.a:hover {
    background-color: green;
    border-radius: 8px;
}

.b {
    border: 2px solid blue;
}
```

Reply:
thanks, it would still be ok if the last one is missing ` ;`